classes/author/Collector.inc.php

```diff
@@ -179,12 +204,24 @@ public function getQuery()
                 });
         }
 
-        $q->orderBy('a.seq', 'ASC');
+        if (!empty($this->includeInBrowse)) {
```

Comment:
This should be changed to a strict `!== false` test as part of tightening up behaviour around "falsy" data. (Here and below)

Reply:
That was a copy-paste error from the methods that use an array. I have changed it to `isset` for all.

Reply:
Can you ping me when these changes are pushed up?